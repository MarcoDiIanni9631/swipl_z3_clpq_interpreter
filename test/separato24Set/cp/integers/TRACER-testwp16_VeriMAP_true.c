# 1 "MAP/SAFE-exbench/TRACER-testwp16.tmp.c"
# 1 "<command-line>"
# 1 "MAP/SAFE-exbench/TRACER-testwp16.tmp.c"
# 21 "MAP/SAFE-exbench/TRACER-testwp16.tmp.c"
main(){

  int t,x2,x3,temp;


  t = 1;
  x2 = temp;
  x3 = temp;



  if (x2 == 0){

    if (x3 == 3)
      t++;
  }


  t++;

  __VERIFIER_assert(!(t>1000));

}
