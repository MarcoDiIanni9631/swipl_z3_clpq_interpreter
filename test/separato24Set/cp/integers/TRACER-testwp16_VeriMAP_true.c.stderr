MAP/SAFE-exbench/TRACER-testwp16.tmp.c:41: Warning: Body of function main falls-through. Adding a return statement
